### PR TITLE
Add support to configure extra labels

### PR DESCRIPTION
**Pull request by kasparjarek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,3 +22,17 @@ saramaprom.ExportMetrics(metricRegistry, saramaprom.Options{})
...
 
 `saramaprom.Options` enables you to for example configure how often the metrics are refreshed, namespace of the metrics, etc.
+
+Multiple metric registries can be defined to not mix values of several Kafka cluster.
+But in that case the prometheus metrics have to be distinguished by labels. Use `ExtraLabels` option for that:
+```
+// create new registry and export metrics for cluster Foo 
+configFoo := sarama.NewConfig()
+configFoo.MetricRegistry = metrics.NewRegistry()
+saramaprom.ExportMetrics(configFoo.MetricRegistry, saramaprom.Options{ExtraLabels: map[string]string{"cluster": "foo"},})
```
Comment (basovnik) [inline README.md]: Minor: Is the comma (`,`) at the end of line required? I would remove it. The same below.

File: saramaprom.go [hunk 2]
```diff
@@ -20,6 +20,9 @@ type Options struct {
 	Namespace string
 	Subsystem string
 
+	// Extra labels to be added to all metrics. Beware, this will override the default label value if configured with same name.
```
Comment (basovnik) [inline saramaprom.go:23]: Minor: What about to add a unit test where the extra label would override a label value with the same name?

**Review by basovnik — APPROVED** [review]

**Review by rschone — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kasparjarek) [reply README.md]: removed

File: saramaprom.go (hunk 2 above)
Comment (kasparjarek) [reply saramaprom.go:23]: I don't think it's needed. It's an edge case and I don't expect anyone to configure it that way :)

**Pull request merged**